### PR TITLE
ANDROID-10672 Fix corner radius in boxed inverse list row item

### PR DESCRIPTION
**Pull request by yamal-alm**
### :goal_net: What's the goal?
Corner radius on  boxed list row items should be set to 8dp, it was changed for its standard version but not for its "inverse" version.

### :construction: How do we do it?
4dp to 8dp on boxed inverse list row item

### ☑️ Checks
- [x] I updated the documentation (readmes, wikis, etc). If no need to update documentation explain why.
- [x] Tested with dark mode.
- [x] Tested with API 21.

### :test_tube: How can I test this?
- [x] 🖼️ Screenshots/Videos
- [x] [Mistica App download link](https://install.appcenter.ms/orgs/tuenti-organization/apps/mistica-catalog/distribution_groups/public)
Captura de pantalla 2022-07-11 a las 15 17 19

- [x] Reviewed by Mistica design team

XML
Captura de pantalla 2022-07-11 a las 15 11 17

Compose
Captura de pantalla 2022-07-11 a las 15 10 52

### CODE REVIEW

File: library-compose/src/main/java/com/telefonica/mistica/compose/list/ListRowItem.kt [hunk 1]
```diff
@@ -71,17 +73,12 @@ fun ListRowItem(
...
                 color = MisticaTheme.colors.border,
-                shape = RoundedCornerShape(8.dp),
+                shape = RoundedCornerShape(B0XED_CORNER_RADIUS),
             )
         BackgroundType.TYPE_BOXED_INVERSE -> Modifier
             .background(
                 color = MisticaTheme.colors.backgroundBrand,
-                shape = RoundedCornerShape(4.dp),
-            )
-            .border(
```
Comment (yamal-alm) [inline library-compose/src/main/java/com/telefonica/mistica/compose/list/ListRowItem.kt:81]: fixing compose version of this componente: grey border should not be included on boxed inverse list row items

**Review by manolovn — APPROVED** [review]
thx!

**Review by AnaMontes11 — APPROVED** [review]
Nice!

**Review by DevPabloGarcia — COMMENTED** [review]
Nice1

**Pull request merged**